src/stc/stc.interface.h.in

```diff
@@ -1087,7 +1087,7 @@ public:
 
     @c wxEVT_STC_INDICATOR_RELEASE
 
-    - Generated when a click over a indicator has been released.
+    - Generated when a click-over indicator has been released.
```

Comment:
Sorry, what does "a click-over indicator" actually mean? Are you sure the dash here is really needed? I.e. isn't this supposed to be a "click (== mouse button press) over an indicator" instead?

Reply:
Oh, I misread that, sorry. I just updated the PR to fix that.